scripts/rook.sh

```diff
@@ -86,8 +103,8 @@ function check_ceph_cluster_health() {
 	for ((retry = 0; retry <= ROOK_DEPLOY_TIMEOUT; retry = retry + 5)); do
 		echo "Wait for rook deploy... ${retry}s" && sleep 5
 
-		CEPH_STATE=$(kubectl -n rook-ceph get cephclusters -o jsonpath='{.items[0].status.state}')
-		CEPH_HEALTH=$(kubectl -n rook-ceph get cephclusters -o jsonpath='{.items[0].status.ceph.health}')
+		CEPH_STATE=$(kubectl_retry -n rook-ceph get cephclusters -o jsonpath='{.items[0].status.state}')
```

Comment:
the output on stdout is consumed here, if it is not 'clean' but contains error messages from failed retries, the `kubectl_retry` function needs to capture the stdout/stderr output of each kubectl call.